graphql/index.js

```diff
@@ -1,12 +1,14 @@
 import Aggregator from "./aggregator";
 import Getter from "./getter";
 import Explorer from "./explorer";
+import Raw from "./raw";
 
 const graphql = (client) => {
   return {
     get: () => new Getter(client),
     aggregate: () => new Aggregator(client),
     explore: () => new Explorer(client),
+    raw: (query) => new Raw(client, query),
```

Comment:
query should be added by `withQuery` method, not given in the constructor same as parameters are handled in other clients